<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Centralna linija</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Boja linije : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Izgled linije : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Sirina linije : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="stilMidline"/>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="sirinaMidline">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="bojaMidline">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Prozirnost : </string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="alphaMidline">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Dobar SPAN marker</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Velicina markera : </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Boja markera : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Izgled markera : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="okMarker"/>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="okSize">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="okBoja">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Prozirnost : </string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="okAlpha">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Los SPAN marker</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Boja markera : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Velicina markera : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Izgled markera : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="badMarker"/>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="badSize">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="badBoja">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Prozirnost : </string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="badAlpha">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Granicna linija</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Izgled linije : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Izbor crtanja : </string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Boja linije : </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Sirina linije : </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="granicaLine">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="granicaSirina">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="granicaCheck">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="granicaBoja">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Prozirnost : </string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="granicaAlpha">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Fill izmedju granicnih linija</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Boja sjencanja : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Izbor crtanja : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="fillCheck">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="fillBoja">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Prozirnost : </string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="fillAlpha">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
